test/utils/fizzy_engine.cpp

```diff
@@ -72,9 +73,54 @@ bytes_view FizzyEngine::get_memory() const
     return {m_instance->memory->data(), m_instance->memory->size()};
 }
 
-std::optional<WasmEngine::FuncRef> FizzyEngine::find_function(std::string_view name) const
+namespace
 {
-    return fizzy::find_exported_function(m_instance->module, name);
+FuncType translate_signature(std::string_view signature)
+{
+    std::string_view inputs(signature.begin(), signature.find(":"));
+    std::string_view outputs(inputs.end() + 1);  // Is this a terrible hack?
+
+    FuncType func_type;
+    for (auto input : inputs)
+    {
+        if (input == 'i')
```

Comment:
don't repeat these `if`s, make a helper `fizzy::ValType charToValType(char)`

Reply:
Feels like an overkill to be honest.

Reply:
Did it anyway.